src/test/simple_test.hpp

```diff
@@ -0,0 +1,25 @@
+
+#include "include_libsnark.hpp"
+
+namespace libzeth
+{
+namespace test
+{
+
+/// Generate a simple test circuit with 1 public input 'y' and auxiliary
```

Comment:
Let's stick to:
```
/*
 *
**/
```

For comments on multiple lines, rather than `///` which are not used anywhere else in the code base